employer-engagement/conda_dependencies.yml

```diff
@@ -0,0 +1,15 @@
+name: employer_engagement_env
+dependencies:
+  - python=3.6.2
```

Comment:
Is a 2017 released version of Python needed as a dependency or can a more up-to-date version be used?

Reply:
Hi Jack, no dependency on an older version, purely from the original sample code supplied to us.  Happy to update.  Is there a latest version which is supported by DevOps or simply the latest version?

Reply:
There is no 'supported by DevOps' here, it will pull whatever version you want to specify.

Reply:
Updated to latest version of python